public/scss/style.scss

```diff
@@ -2026,3 +2026,7 @@ $palette-red: $palette4;
         pointer-events: none;
     }
 }
+
+code {
+    white-space: preserve !important;
```

Comment:
`!important` is best to avoid imo.

Reply:
Agreed. In this case it doesn't work without it since the rule is already set by UIkit.

Reply:
You can at least make it more specific? If a rule is more specific than another rule, it overrides it anyway.